packages/aws-sdk-signers/pyproject.toml

```diff
@@ -34,8 +34,8 @@ exclude = [
  "tests",
 ]
 
-[project.optional-dependencies]
-test = [
+[dependency-groups]
+dev = [
```

Comment:
Was there a reason we're using `dev` here? It looks like most examples and our generated code above are using `test` or `tests`. Ideally we want to align a single consistent identifier.

Reply:
I initially thought `dev` was was best since it gets installed by default with `uv` sync. However, we can achieve similar behavior by adding more specific groups to the default groups list like: `default-groups = ["test", "lint", "typing"]`.

As for naming, I'm going to go align with what's used in the (PEP 735 examples](https://peps.python.org/pep-0735/#example-dependency-groups-table).

Reply:
Updates are reflected in 10b4f2fece1c40d698648f3120b6eeeb7da7d5b3